### ISSUE TITLE
Inference with masked/cencored data?

### ISSUE THREAD

**Issue by bfredl** [blei-lab/edward#669]
Sorry if newbie question, I searched the repo but couldn't find anything about "mask" or "censor".  We are training models with masked/missing data, by discarding the missing data's contribution to the likelihood function, like `loss = scale*(mask*(network_output-y)**2).sum()`. What would be the best way to do this in edward?

**Comment by dustinvtran** [comment]
Check out and feel free to post comments in https://discourse.edwardlib.org/t/how-to-handle-missing-values-in-gaussian-matrix-factorization/95/2. In general, we recommend use-related questions on the [Forum](https://discourse.edwardlib.org).

Depending on your use case, another useful argument to look at is `scale` in [`inference.initialize()`](http://edwardlib.org/api/inference.html#edward.inferences.Inference). You can use scale to zero out specific variables via a masking operation.

**Comment by bfredl** [comment]
`scale` was what I was looking for, didn't realize from examples it can apply to all dimensions. Thanks!
